### PR TITLE
added signup button

### PR DESCRIPTION
**Pull request by lknair2015**

### CODE REVIEW

File: crossroads.net/app/camps/camps_family/camps_family.controller.js [hunk 1]
```diff
@@ -13,6 +13,13 @@ class CampsFamilyController {
   isSignedUp(member) {
     return member.signedUpDate !== null;
   }
-}
 
+  divClass(member) {
```
Comment (silbermm) [inline crossroads.net/app/camps/camps_family/camps_family.controller.js]: not sure how this is working... a class method `isSignedUp()` has to be referenced by `this`

File: crossroads.net/app/camps/camps_family/camps_family.html [hunk 2]
```diff
@@ -1,9 +1,18 @@
...
-    <li class="list-group-item" data-ng-repeat="member in $ctrl.campsService.family">
-      {{member.preferredName}} {{member.lastName}}
-      <span class="label label-warning" data-ng-if="!member.isEligible && !$ctrl.isSignedUp(member)">Not Eligible</span>
-      <span class="label label-success" data-ng-if="$ctrl.isSignedUp(member)"> Signed Up on {{member.signedUpDate | date : "M/dd/yyyy" }} </span>
-    </li>
-  </ul>
+    <ul class="list-group col-sm-10 col-md-8">
+      <li class="list-group-item hard-sides hard-ends" data-ng-repeat="member in $ctrl.campsService.family">
+        <div class="clearfix">
+          <div class="col-xs-12 col-top col-sm-9 col-md-10 soft-half" ng-class='divClass(member)'>
```
Comment (silbermm) [inline crossroads.net/app/camps/camps_family/camps_family.html]: `divClass` has to be referenced by `$ctrl`. This is probably not working.

File: crossroads.net/app/camps/camps_family/camps_family.html [hunk 3]
```diff
@@ -1,9 +1,18 @@
...
+            <span ng-class='pClass(member)' > {{member.preferredName}} {{member.lastName}} </span>
```
Comment (silbermm) [inline crossroads.net/app/camps/camps_family/camps_family.html]: `pClass` has to be referenced by `$ctrl`. There is now pClass function in the controller though.

File: crossroads.net/app/camps/camps_family/camps_family.html [hunk 4]
```diff
@@ -1,9 +1,18 @@
...
+            <span class="label label-warning" data-ng-if="!member.isEligible && !$ctrl.isSignedUp(member)">Not Eligible</span>
+            <span class="label label-success" data-ng-if="$ctrl.isSignedUp(member)"> Signed Up on {{member.signedUpDate | date : "M/dd/yyyy" }} </span>
+          </div>
+          <div class="col-xs-12 col-sm-3 col-md-2 hard-sides mobile-soft-half-sides mobile-push-half-ends">
+            <a href="#" data-ng-if="$ctrl.isSignedUp(member)" disabled class="table-btn btn-standard disabled">Sign Up </a>
+            <a href="#" ng-if="!member.isEligible && !$ctrl.isSignedUp(member)" disabled class="table-btn btn-standard disabled">Sign Up</a>
+            <a ui-sref="campsignup.camper({camperId: member.contactId})" ng-if="!$ctrl.isSignedUp(member) && member.isEligible" class="table-btn btn-standard">Sign Up</a>
```
Comment (silbermm) [inline crossroads.net/app/camps/camps_family/camps_family.html]: don't you also have to pass the campId to the route?

Comment (lknair2015) [inline crossroads.net/app/camps/camps_family/camps_family.html]: the current state has camp id. so we don't need to pass it

File: crossroads.net/app/camps/camps_family/camps_family.controller.js (hunk 1 above)
Comment (lknair2015) [inline crossroads.net/app/camps/camps_family/camps_family.controller.js]: updated it

File: crossroads.net/app/camps/camps_family/camps_family.html (hunk 2 above)
Comment (lknair2015) [inline crossroads.net/app/camps/camps_family/camps_family.html]: updated it

File: crossroads.net/app/camps/camps_family/camps_family.html (hunk 3 above)
Comment (lknair2015) [inline crossroads.net/app/camps/camps_family/camps_family.html]: updated it

**Pull request merged**